_RFCs/0001-rfc-for-rfcs.md

```diff
@@ -0,0 +1,290 @@
+marked as such because we want neither to think about evaluating the proposal
+nor about implementing the described feature until some time in the future, and
+we believe that we can afford to wait until then to do so. Postponed pull
+requests may be re-opened when the time is right. We don't have any formal
+process for that, you should ask maintainers in the relevant slack channel.
+
+Usually an RFC pull request marked as "postponed" has already passed an
+informal first round of evaluation, namely the round of "do we think we would
+ever possibly consider making this change, as outlined in the RFC pull request,
+or some semi-obvious variation of it." (When the answer to the latter question
+is "no", then the appropriate response is to close the RFC, not postpone it.)
+
+
+### Help this is all too informal!
+[Help this is all too informal!]: #help-this-is-all-too-informal
+
+The process is intended to be as lightweight as reasonable for the present
+circumstances. As usual, we are trying to let the process be driven by
+consensus and community norms, not impose more structure than necessary.
+
+
+[#core-plans slack channel]: https://habitat-sh.slack.com/
+[Core-Plans RFC issue tracker]: https://github.com/habitat-sh/core-plans/labels/C-RFC
+
+## Attribution
+[Attribution]: #attribution
+
+THIS DOCUMENT IS HEAVILY INFLUENCED AND MANY OF THE WORDS ARE TAKEN DIRECTLY FROM THE RUST RFC PROCESS. [Rust RFCs](https://github.com/rust-lang/rfcs)
+
+
+#################
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+It's possible that in implementing an RFC process we'll end up with more project transparency which could result in an uptick in contribution which in turn could cause the core-plans maintainers to be further inundated with work.
+
+# Rationale and alternatives
+[alternatives]: #alternatives
+
+Most open and free software projects already use an RFC process to get transparency over development and maintenance happening within a project as well as giving contributors perspective over project direction. This pattern has been adopted by hundreds of thousands of FOSS projects. This change will provide our community another feedback mechanism as well as a way to teach new contributors the ins-and-outs of core-plans development.
+
+The alternative is to not have an RFC process. We don't want to go this path as having one will help to shore up behaviors and stability in the core set of packages that all Habitat users will require going forward.
+
+The impact of not adopting an RFC process could mean continued low-quality of various packages in core as well as the inability to lean on governance to disallow poor quality merges. It can also hinder our contribution levels if community members feel that they can't safely work on something and get it merged into the core packages in a reasonable amount of time.
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+Is 10 days too long/too short of an FCP?
```

Comment:
10 calendar days, or 14 calendar days - i.e. 2 business weeks?

Reply:
10 Calendar days

Reply:
I feel that 14 calendar days allows for a little more time to think and consider items under review. Also, it's an excuse to use the term "fortnight".

Reply:
Yeah I think that's the right thing. I'll go ahead and update this.